### ISSUE TITLE
Problems on debian buster

### ISSUE THREAD

**Issue by RincewindsHat** [NETWAYS/puppet-icinga-installer#2]
Fresh debian 10 installation

```
 # icinga-installer --scenario server
2021-03-14 13:39:50 [NOTICE] [root] Loading default values from puppet modules...
Puppet 5.5.10 does not meet requirements for puppetlabs-apache (>= 6.0.0 < 8.0.0)
Cannot continue due to incompatible version of Puppet. Use --skip-puppet-version-check to disable this check.
2021-03-14 13:39:52 [ERROR ] [root] Puppet 5.5.10 does not meet requirements for puppetlabs-apache (>= 6.0.0 < 8.0.0)
2021-03-14 13:39:52 [ERROR ] [root] Incompatible version of Puppet used, cannot continue
```

**Comment by lbetz** [comment]
Did you use the package? There is missed a dependency to puppet 6.

**Comment by RincewindsHat** [comment]
Yes, just tested the debian package

**Comment by lbetz** [comment]
Tomorrow I'll have a look to the puppet modules, maybe it's possible to use versions to use also puppet 5.

**Comment by lbetz** [comment]
refs #3

**Comment by lbetz** [comment]
I release 0.3.0 of icinga-installer, the package are available on https://packages.netways.de/extras.

I fixed the requirements of puppet-agent to >= 5.5.10 < 7, so it works on Buster out of the box without the puppet repository.
